### PR TITLE
Bokeh url

### PR DESCRIPTION
**Pull request by svecon**
"0.0.0.0 is the invalid, un-routable address." on Windows
https://news.ycombinator.com/item?id=18978357

### CODE REVIEW

**Comment by svecon** [comment]
Not sure why those two previous commits (which were merged already) still show in this PR.

**Comment by Bachibouzouk** [comment]
> Not sure why those two previous commits (which were merged already) still show in this PR.

Probably because you are out of date with master

**Comment by janpipek** [comment]
I agree with the defaults - on Windows, I also had to change 0.0.0.0 to 127.0.0.1.

On the other hand, are you sure that changing the docker & deployment configuration does not break anything? /cc @hnykda

**Comment by hnykda** [comment]
No, it shouldn't break anything, I tried to do it reasonably well, but didn't test on windows (I have linux and mac)

**Comment by hnykda** [comment]
This won't work in docker-compose (the container localhost is not seen from the host). So I would still keep it to 0.0.0.0 inside docker-compose. Can we?

File: deploy/chart/templates/deployment-web.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
           - uvicorn
           - main:app
           - --host
-          - 0.0.0.0
+          - 127.0.0.1
```
Comment (hnykda) [inline deploy/chart/templates/deployment-web.yaml]:
```suggestion
          - 0.0.0.0
```

File: deploy/chart/templates/deployment-charts.yaml [hunk 2]
```diff
@@ -56,7 +56,7 @@ spec:
             - --port
             - "5001"
             - --address
-            - 0.0.0.0
+            - 127.0.0.1
```
Comment (hnykda) [inline deploy/chart/templates/deployment-charts.yaml]:
```suggestion
            - 0.0.0.0
```

**Pull request merged**